content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -90,19 +90,39 @@ Renders to:
 
 ## Glossary
 
+There are two glossary tooltips.
```

Comment:
This is confusing ... we don't count tooltips. How about revise it as kbhawkey suggested:

```
There are two glossary related shortcodes: `glossary_tooltip` and `glossary_definition`.
```

Reply:
Good point. Revised.